go_lib/dependency/helm/helm.go

```diff
@@ -93,7 +94,7 @@ func WithTimeout(timeout time.Duration) Option {
 	}
 }
 
-func (client *helmClient) Upgrade(releaseName string, templates, values map[string]interface{}, _ bool) error {
+func (client *helmClient) Upgrade(releaseName, releaseNamespace string, templates, values map[string]interface{}, _ bool, pr ...postrender.PostRenderer) error {
```

Comment:
I have tried to. But it does't work, because HelmClient is getting from the cache during the hook execution. And pointers to the PostRenderer becomes different.
We have to remove cache or pass this PostRenderer in the function. I decided to use function call at the moment